kolibri/plugins/learn/assets/test/views/topic-content.spec.js

```diff
@@ -192,7 +192,12 @@ describe('TopicsContentPage', () => {
         beforeEach(async () => {
           useDownloadRequests.mockImplementation(() =>
             useDownloadRequestsMock({
-              isDownloadedByLearner: () => true,
+              downloadRequestMap: {
+                [CONTENT_ID]: {
+                  id: CONTENT_ID,
+                  status: 'COMPLETED',
```

Comment:
this is the sort of thing in a test that I need to remember for some future date, that I can actually put in a test